aws/resource_aws_wafv2_web_acl_logging_configuration.go

```diff
@@ -34,10 +38,65 @@ func resourceAwsWafv2WebACLLoggingConfiguration() *schema.Resource {
 				Description: "AWS Kinesis Firehose Delivery Stream ARNs",
 			},
 			"redacted_fields": {
-				Type:        schema.TypeSet,
-				Optional:    true,
-				MaxItems:    100,
-				Elem:        wafv2FieldToMatchBaseSchema(),
+				// To allow this argument and its nested fields with Empty Schemas (e.g. "method")
+				// to be correctly interpreted, this argument must be of type List,
+				// otherwise, at apply-time a field configured as an empty block
+				// (e.g. body {}) will result in a nil redacted_fields attribute
+				Type:             schema.TypeList,
+				Optional:         true,
+				DiffSuppressFunc: suppressRedactedFieldsDiff,
```

Comment:
understandably not an ideal side-effect  :/ any thoughts on using a different hack to work around the order diffs?